cmd/keygenerator/CLI.md

```diff
@@ -17,9 +17,8 @@ AUTHOR:
 GLOBAL OPTIONS:
    --num-keys value  How many keys should generate. Example: 1 (default: 1)
    --key-type value  What king of keys should generate. Available options: validator, wallet, both (default: "validator")
-   --help, -h        show help
-   --version, -v     print the version
-   
-
+   --help, -h        Show help
+   --version, -v     Print the version
+   --hex-key-prefix  Value special pattern for the hex key prefix: Example 00f00 (default nopattern)
```

Comment:
I think that exactly this pattern will not work, because it should have an even number of hexadecimal digits, right?